guides/v2.2/extension-dev-guide/view-models.md

```diff
@@ -0,0 +1,62 @@
+---
+title: View models
+contributor_name: Space 48
```

Comment:
Add the following line to the frontmatter. `title` and `group` are always required  in the main repo

group: php-developer-guide